Makefile

```diff
@@ -377,6 +380,11 @@ ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu")
     ALTIVEC_CFLAGS = -maltivec
     HAVE_VM_COMPILED=true
   endif
+  ifeq ($(ARCH),ppc64le)
+    OPTIMIZE += -mcpu=power8
+    ALTIVEC_CFLAGS = -maltivec
+    HAVE_VM_COMPILED=false
```

Comment:
Yes, this is deliberate. I will make another Pull Request to fix the `vm_powerpc` code to make it work with ppc64le